### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by AbdelazizSoliman**
In this pull request:

- [ ] I created add and delete function.
- [ ] saved to local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+const input = document.querySelector('#new-task-input');
+const submitButton = document.querySelector('.add');
+const taskList = document.querySelector('#tasks');
+let tasks = [];
+
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+/*eslint-disable*/
```
Comment (anagudelogu) [inline src/index.js]: - [ ] Please avoid disabling linter rules as they ensure you follow a consistent code format and also apply best practices, instead consider fixing the errors! 🤓 🔧

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+loadTasks();
+
+// Add task
+submitButton.onclick = function (e) {
+  e.preventDefault();
+  if (input.value !== "") {
+    /*eslint-disable*/
```
Comment (anagudelogu) [inline src/index.js]: - [ ] Please avoid disabling linter rules as they ensure you follow a consistent code format and also apply best practices, instead consider fixing the errors! 🤓 🔧

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    createNewTask(input.value);
+    input.value = "";
+  }
+};
+
+taskList.addEventListener("click", (e) => {
+  if (e.target.classList.contains("del")) {
+    const { id } = e.target.parentElement.dataset;
+    /*eslint-disable*/
```
Comment (anagudelogu) [inline src/index.js]: - [ ] Please avoid disabling linter rules as they ensure you follow a consistent code format and also apply best practices, instead consider fixing the errors! 🤓 🔧

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+// Add task
+submitButton.onclick = function (e) {
+  e.preventDefault();
+  if (input.value !== "") {
+    /*eslint-disable*/
+    createNewTask(input.value);
+    input.value = "";
+  }
+};
```
Comment (anagudelogu) [inline src/index.js]: - [ ] According to the requirements:
image

This new functionality should be in a new, separate file, please consider applying it! 🤓

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    const tasksFromStorage = JSON.parse(data);
+    tasksFromStorage.forEach(addTaskToDOM);
+    tasks = tasksFromStorage;
+  }
+}
+
+function deleteTask(taskId) {
+  tasks = tasks.filter((task) => task.id != taskId);
+  saveTasksToLocalStorage();
+}
```
Comment (anagudelogu) [inline src/index.js]: - [ ] Also, according to the requirements: 

image

Indexes should get updated upon deleting a task, but you haven't implemented that feature yet, please consider doing it! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @AbdelazizSoliman, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

- [ ] 🚧 According to the requirements, the edit task feature should be implemented:

image

Please consider doing so! 🤓 🔧 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @AbdelazizSoliman 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**